tools/atmos_bench/run_headless.sh

```
@@ -0,0 +1,80 @@
+# down when done (see atmos_benchmark.dm, ATMOS_HEADLESS_BENCH section).
+#
+# Usage: tools/atmos_bench/run_headless.sh <tag> [map] [skip-build] [cycles]
+#   tag        label for the result file (e.g. baseline, fix1)
+#   map        json name from _maps/, default icemoonstation
+#   skip-build pass "skip-build" to reuse the existing tgstation.dmb
+#   cycles     SSair cycles to run, default 240 (compile-time default)
+#
+# Result: tools/atmos_bench/results/<timestamp>_<tag>.jsonl
+# Analyze: python tools/atmos_bench/analyze.py <file> [file2 ...]
+set -u
+cd "$(dirname "$0")/../.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)